tests/components/vizio/conftest.py

```diff
@@ -174,7 +176,7 @@ def vizio_update_fixture():
     ), patch(
         "homeassistant.components.vizio.media_player.VizioAsync.get_all_settings",
         return_value={
-            "volume": int(MAX_VOLUME[DEVICE_CLASS_SPEAKER] / 2),
+            "volume": int(MAX_VOLUME[MediaPlayerDeviceClass.SPEAKER] / 2),
```

Comment:
This change is wrong - the import came from `pyvizio` and should continue to do so.

Reply:
Good spot, agreed - done.